INSTALL_SCONS.md

```diff
@@ -0,0 +1,677 @@
+Installation Using SCons
+========================
+
+Prior to version 2.0, this was the only installation method for installing
+GalSim.  It is still supported, mostly in case some users have trouble with
+the setup.py method, but not recommended for most users.
+
+Please see the instructions in INSTALL.md first to see if that method
+will work for you.
+
+
+1. Software required before building GalSim
+===========================================
```

Comment:
I think the numbering is what messed it up.  It looks like it tried to make a numbered list rather than treat it as a heading.  Probably an ambiguity in Markdown's specs and GitHub's implementation changed at some point since we first wrote this.